examples/structured_data/classification_with_grn_and_vsn.py

```diff
@@ -0,0 +1,413 @@
+"""
+Title: Classification with Gated Residual and Variable Selection Networks
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2021/01/15
+Last modified: 2021/01/15
+Description: Using Gated Residual and Variable Selection Networks for income level prediction.
```

Comment:
Suggestion: comments similar to https://github.com/keras-team/keras-io/pull/341#discussion_r548923687 and https://github.com/keras-team/keras-io/pull/341#discussion_r548924150 regarding the title and description.